### ISSUE TITLE
ceedling -T not working correctly

### ISSUE THREAD

**Issue by wolf99** [ThrowTheSwitch/Ceedling#222]
I've recently started using the most recent Ceedling where before I had been using an old version (not sure of the Ceedling version number but Ruby was 2.3.1). 

In the old version `rake -T` works just as intended. In the current version however, `ceedling -T` instead runs something apparently equivalent to `ceedling test:all`

**Comment by mvandervoord** [comment]
Does the documentation still say -T somewhere? We've switched to a more understandable `ceedling help` in the latest versions.

**Comment by wolf99** [comment]
Ah I see `ceedling help` now 👍 . But yes, `ceedling -T` is still mentioned a few times under the `Now What? How Do I Make It GO?` section of CeedlingPacket

**Comment by mvandervoord** [comment]
Thanks. I'll fix that. :)

**Comment by dpostorivo** [comment]
IMO, if it's easy we should keep `-T` since it's a rake idiom that people familiar with it default to. I know we are moving away from thinking in terms of rake and thinking more of ceedling as standalone, but if it causes less confusion to keep it then we should keep it.

**Comment by wolf99** [comment]
I don't think anything else in ceedling currently conflicts with `-T`, that I can think of off the top of my head; would it hurt to have both? (both `help` and `-T` that is)

**Comment by dpostorivo** [comment]
Yep I'd also prefer to have both (sorry if it wasn't obvious in my previous reply).

**Comment by Miv13** [comment]
I just happened to be in the neighbourhood and wanted to point out that as for now (ceedling v. 0.28.3) the description of `ceedling -T` (e.g. in `CeedlingPacket.md`) is still as follows:

"List all available Rake tasks with descriptions (Rake tasks without descriptions are not listed). -T is a command line switch for Rake and not the same as tasks that follow."

...and the functionality is still "run all tests". ;)
